### PR TITLE
 [OYPD-638] Improve theming of Lily theme (camp header, news page)

### PR DESCRIPTION
**Pull request by anprok**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!
https://docs.google.com/document/d/176rGvTaJujv7yNix5u-egOnLEBnmJBNRmzimnaqjjWE/edit 
items 3,4
## Steps for review

- [x] open any camp and check that header is available
- [x] open /news path, check style

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build343 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build343 |
| Installation process | http://install.openy-dev.ffwua.com/build343/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER_COMPOSER/343/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build343/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build343/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build343/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build343/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build343/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL http://ci.openymca.org/build343/profiles/contrib/openy/build/reports/behat and failed screenshots http://ci.openymca.org/build343/profiles/contrib/openy/build/reports/behat/screenshots

File: themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig [hunk 1]
```diff
@@ -80,6 +80,150 @@
...
+              <span class="plaintext">
+                {{ address|striptags }}
+              </span>
+              <a class="directions" target="_blank" href="{{ content.directions_url }}">
+                {{ 'Directions'|t }}
+                <span class="glyphicon glyphicon-share"></span>
+              </a>
+            </div>
+            <div class="hours col-lg-4 col-md-4 col-sm-4 clearfix">
+              {% if node.bundle == 'branch' %}
```
Comment (ddrozdik) [inline themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig]: I think this condition does not make sense here.

File: themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig [hunk 2]
```diff
@@ -80,6 +80,150 @@
...
+              </span>
+              <br>
+              <a class="directions" target="_blank" href="{{ content.directions_url }}">
+                {{ 'Directions'|t }}
+                <span class="glyphicon glyphicon-share"></span>
+              </a>
+            </div>
+            <hr>
+            <div class="hours col-lg-4 col-md-4 col-sm-4 clearfix">
+              {% if node.bundle == 'branch' %}
```
Comment (ddrozdik) [inline themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig]: I think this condition does not make sense here.

File: themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig [hunk 3]
```diff
@@ -80,6 +80,150 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="location-details masthead-menu">
+      <div class="container">
+        <div class="row">
+          <div class="col-xs-12">
+            <button type="button" class="btn btn-link location-hours-toggle" data-toggle="collapse" data-target="#directions-hours-mobile">
+              <span class="glyphicon glyphicon-chevron-down pull-right"></span>
+              {% if node.bundle == 'branch' %}
```
Comment (ddrozdik) [inline themes/openy_themes/openy_lily/templates/node/node--camp--full.html.twig]: I think this condition does not make sense here.

**Review by ddrozdik — CHANGES_REQUESTED** [review]

**Review by ddrozdik — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build344 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build344 |
| Installation process | http://install.openy-dev.ffwua.com/build344/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER_COMPOSER/344/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build344/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build344/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build344/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build344/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build344/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL http://ci.openymca.org/build344/profiles/contrib/openy/build/reports/behat and failed screenshots http://ci.openymca.org/build344/profiles/contrib/openy/build/reports/behat/screenshots

**Pull request merged**